### ISSUE TITLE
[Bug]: smdk generate fails

### ISSUE THREAD

**Issue by adfaure** [infinyon/fluvio#3921]
**What happened**
I am trying the quickstart guide, and at the [step](https://www.fluvio.io/docs/#build-a-filter-smartmodule) smdk fails at generating a smartmodule.

```bash
RUST_BACKTRACE=1 smdk generate quickstart                                                        nix-shell-env
Generating new SmartModule project: quickstart
🤷   Please set a group name: adfaure
✔ 🤷   Which type of SmartModule would you like? · filter
✔ 🤷   Will your SmartModule use init parameters? · false
✔ 🤷   Will your SmartModule be public? · false
[1/7]   Done: .gitignore
[2/7]   Done: Cargo.toml
[3/7]   Done: README.md
[4/7]   Done: SmartModule.toml
[5/7]   Done: rust-toolchain.toml                                                                                [6/7]   Done: src/lib.rs                                                                                         [7/7]   Done: src                                                                                                thread 'main' panicked at /home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.34.0/src/time/interval.rs:138:26:
there is no reactor running, must be called from the context of a Tokio 1.x runtime
stack backtrace:
   0: rust_begin_unwind
             at /rustc/07dca489ac2d933c78d3c5158e3f43beefeb02ce/library/std/src/panicking.rs:645:5
   1: core::panicking::panic_fmt
             at /rustc/07dca489ac2d933c78d3c5158e3f43beefeb02ce/library/core/src/panicking.rs:72:14
   2: tokio::runtime::scheduler::Handle::current::panic_cold_display
   3: tokio::time::sleep::Sleep::new_timeout
   4: tokio::time::interval::interval
   5: hyper::client::pool::PoolInner<T>::put
   6: <hyper::client::pool::Pooled<T> as core::ops::drop::Drop>::drop
   7: core::ptr::drop_in_place<hyper::client::pool::Pooled<hyper::client::client::PoolClient<hyper::body::body::Body>>>
   8: hyper::client::client::Client<C,B>::retryably_send_request::{{closure}}
   9: <hyper::client::client::ResponseFuture as core::future::future::Future>::poll
  10: fluvio_future::http_client::send::{{closure}}
  11: fluvio_hub_util::hubaccess::HubAccess::get_action_auth::{{closure}}
  12: async_std::task::task_locals_wrapper::TaskLocalsWrapper::set_current
  13: <futures_lite::future::Or<F1,F2> as core::future::future::Future>::poll
  14: async_io::driver::block_on
  15: async_global_executor::executor::block_on
  16: async_std::task::task_locals_wrapper::TaskLocalsWrapper::set_current
  17: async_std::task::block_on::block_on
  18: smdk::hub::set_hubid
  19: smdk::generate::GenerateCmd::process
  20: smdk::cmd::SmdkCommand::process
  21: smdk::main
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
```

**Expected behavior**
I would like to have the same result as stated in the quickstart guide.

**Describe the setup**
- I am using fluvio install

**How to reproduce it (as minimally and precisely as possible)**

This one is easy: https://www.fluvio.io/docs/#build-a-filter-smartmodule

**Log output**

I don't think that is relevant here, since I don't start fluvio.

**Environment (please complete the following information):**
- OS: Nixos
- Fluvio Version 0.11.5
- Kubernetes version: I don't have it installed
- Minikube or other k8 version (if used): use `minikube version`

**Additional context**

Instead of using smdk generate, I used my cargo installation and followed the readme instructions and it worked:

```
$ cargo install cargo-generate
$ cargo generate --git https://github.com/infinyon/fluvio-smartmodule-template
```

**Comment by digikata** [comment]
Hi @adfaure,  thanks for trying out fluvio and smdk.

There is an incoming fix for this, but a workaround is to do the following: `smdk generate quickstart --project-group adfaure`

**Comment by adfaure** [comment]
Indeed, it works. Thank you!

It is up to you if you want to close the issue now, or wait for the fix to be pushed.

Regards,
Adrien

**Comment by digikata** [comment]
You're welcome, thanks for verifying the workaround. Let's keep this issue open until the fix is merged.

**Comment by digikata** [comment]
Fixed by #3980
